db/models.py

```diff
@@ -1 +1,30 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE = [("Elf", "Elf"),
+            ("Dwarf", "Dwarf"),
+            ("Human", "Human"),
+            ("Ork", "Ork")]
+    name = models.CharField(max_length=255, unique=True, choices=RACE)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True, null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255, unique=False)
```

Comment:
The 'email' field in the Player model is not set to be unique. If each player should have a unique email address, consider setting 'unique=True'.